templates/client.json.j2

```diff
@@ -2,6 +2,14 @@
   "client": {
     "name": "{{ sensu_client_name }}",
     "address": "{{ ansible_default_ipv4['address'] }}",
-    "subscriptions": {{ sensu_client_subscriptions | to_nice_json }}
+    "subscriptions": {{ sensu_client_subscriptions | to_nice_json(indent=6) }},
```

Comment:
Oh nice, I didn't realize `to_nice_json` took `indent`, I'm going to steal this for some of my other work as well!